### PR TITLE
Fix various CSS issues

### PR DESCRIPTION
**Pull request by silverwind**
- Fix black text being white on base theme
- Fix file/blame button group
- Fix label margin in dropdown (regression from graph pr)
- Fix gitgraph loading on arc-green

Screen Shot 2020-11-30 at 22 17 34
image
Screen Shot 2020-11-30 at 22 33 20

### CODE REVIEW

File: web_src/less/_base.less [hunk 1]
```diff
@@ -1604,10 +1617,6 @@ table th[data-sortt-desc] {
   }
 }
 
-.dropdown .ui.label {
-  margin-left: 0 !important;
-}
-
```
Comment (silverwind) [inline web_src/less/_base.less:1623]: @zeripath I don't suppose you know what this was for? It came from https://github.com/go-gitea/gitea/pull/12766/files#diff-5d1373f9290f762a4751c497b0e8be279eb04caa3a7cdc2fd3f42b9f505a83be. It was affecting the labels in the dashboard dropdown.

Comment (silverwind) [reply web_src/less/_base.less:1623]: ping @zeripath. I don't see anything obvious wrong on the graph dropdown after removing that rule, so I guess we can go ahead with this removal.

Comment (zeripath) [reply web_src/less/_base.less:1623]: cool

**Comment by silverwind** [comment]
This should be ready.

**Review by 6543 — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by 6543** [comment]
:rocket:

**Pull request merged**